roles/cifmw_cephadm/defaults/main.yml

```diff
@@ -87,6 +87,7 @@ cifmw_ceph_rgw_config:
   rgw_max_attr_name_len: 128
   rgw_max_attrs_num_in_req: 90
   rgw_max_attr_size: 1024
+  rgw_keystone_accepted_reader_roles: "SwiftSystemReader"
```

Comment:
Would it make sense to move `rgw_keystone_accepted_reader_roles` up a few line, so it's closer to the `rgw_keystone_accepted_roles` and `rgw_keystone_accepted_admin_roles` keys?

Reply:
mmm yeah, it's more a cosmetic change but it doesn't hurt. I'll move it, so it's easier to read from a visual perspective.

Reply:
Done